app/src/main/java/com/islaharper/dawnofandroid/domain/model/Flavour.kt

```diff
@@ -6,7 +6,7 @@ import com.islaharper.dawnofandroid.util.Constants
 import kotlinx.serialization.Serializable
 
 @Serializable
-@Entity(tableName = Constants.FLAVOUR_DB_TABLE)
+@Entity(tableName = Constants.FLAVOURS_TABLE)
```

Comment:
Can we inline this (if it's only used in one place) or find somewhere more appropriate to put it so its encapsulated?